src/Umbraco.Web.UI.Client/src/views/propertyeditors/grid/dialogs/config.controller.js

```diff
@@ -16,7 +16,23 @@ function ConfigController($scope) {
             $scope.model.close();
         }
     }
-    
+
+    vm.showEmptyState = function() {
+        return $scope.model.config.length === 0
+            && Object.keys($scope.model.config).length === 0
+            && $scope.model.styles.length === 0;
+    }
```

Comment:
@lars-erik is it missing a `Object.keys($scope.model.styles).length === 0` here as well?

Reply:
I never used styles for individual grid editor settings. Dunno if that would ever be a case.  
Could add it for good measure though?